railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_belongs_to_required_by_default.rb

```diff
@@ -1,5 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# Require `belongs_to` associations by default. This is a new Rails 5.0 default,
-# so introduced as a config to ensure apps made with earlier versions of Rails aren't affected when upgrading.
+# Require `belongs_to` associations by default. This is a new Rails 5.0
+# default, so it is introduced as a configuration file to ensure that apps made
```

Comment:
Would change `configuration file` to `configuration option` and also consider changing `configuration` to `config` as that is the term used inside Rails.